site/src/util/mdxComponents.tsx

```diff
@@ -1,10 +1,37 @@
+/* eslint-disable react-hooks/rules-of-hooks */
 import slugify from "slugify";
-import { HTMLAttributes, HTMLProps, ReactNode } from "react";
+import {
+  HTMLAttributes,
+  HTMLProps,
+  ReactNode,
+  useContext,
+  useEffect,
+  useRef,
+} from "react";
 import { TypographyProps, Typography, Box, Paper } from "@mui/material";
 import { Link } from "../components/Link";
 import { InfoCardWrapper } from "../components/InfoCard/InfoCardWrapper";
 import { InfoCard } from "../components/InfoCard/InfoCard";
 import { Snippet } from "../components/Snippet";
+import MDXPageContext from "../components/context/MDXPageContext";
```

Comment:
I generally prefer `StrictCamelCase`, so it’d be `MdxPageContext`. I guess you’ve used all caps in `MDX` here because we have `import { MDXRemoteSerializeResult } from "next-mdx-remote"`.

The new functionality is agnostic of markdown so I guess we can avoid `MDX` / `Mdx` at all.

Reply:
Fixed in https://github.com/blockprotocol/blockprotocol/pull/65/commits/64ff9926ff1067cdcb2abe3705a955fbe2ffab61